administrator/language/en-GB/guidedtours.joomla_welcome_steps.ini

```
@@ -0,0 +1,14 @@
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_TITLE="The menu"
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_DESCRIPTION="<p>The menu provides access to the Administrator management pages. Each section has a collection of pages for managing that particular aspect of Joomla, such as Content, Menus and Users.</p><p>The window pane icons link to individual dashboards for those menu groups. The System menu itself leads to a dashboard.</p>"
+
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_QUICKACCESS_TITLE="The dashboard panels"
```

```suggestion
COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_QUICKACCESS_TITLE="The Dashboard Panels"
```

Follows the style guide

---

Thx